cookbook-efforts/multilingual-prompt-engine/01_create_response_dataset.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
```

Comment:
Could you add an introduction and the steps you follow similarly to here? https://github.com/huggingface/data-is-better-together/blob/main/cookbook-efforts/kto-preference/01_create_preference_task.ipynb

Reply:
In addition, I would encourage you to use [distilabel ](https://github.com/argilla-io/distilabel/tree/main)for data generation if it's not too time-consuming to change the code for you. With it, you can generate data with different models and easily push it to the hub.

Reply:
Great idea! Will use distilabel and add introduction 😄